app/models/grouping.rb

```diff
@@ -229,5 +229,7 @@ def accept_tracker_story
 
     story.current_state = "accepted"
     story.save
+
+    rescue
```

Comment:
Ahah I forgot we added this as a bonus task 😂 

As I forgot, there's a new story in the chores backlog.